src/libraries/System.Globalization/tests/DateTimeFormatInfo/DateTimeFormatInfoFirstDayOfWeek.cs

```diff
@@ -10,209 +10,215 @@ public class DateTimeFormatInfoFirstDayOfWeek
+            yield return new object[] { "nb-NO", DayOfWeek.Monday };
+            yield return new object[] { "no", DayOfWeek.Monday };
+            yield return new object[] { "no-NO", DayOfWeek.Monday };
+            yield return new object[] { "nl-AW", DayOfWeek.Monday };
+            yield return new object[] { "nl-BE", DayOfWeek.Monday };
+            yield return new object[] { "nl-NL", DayOfWeek.Monday };
+            yield return new object[] { "pl-PL", DayOfWeek.Monday };
+            yield return new object[] { "pt-BR", DayOfWeek.Sunday };
+            yield return new object[] { "pt-PT", DayOfWeek.Sunday };
+            yield return new object[] { "ro-RO", DayOfWeek.Monday };
+            yield return new object[] { "ru-RU", DayOfWeek.Monday };
+            yield return new object[] { "sk-SK", DayOfWeek.Monday };
+            yield return new object[] { "sl-SI", DayOfWeek.Monday };
+            yield return new object[] { "sr-Cyrl-RS", DayOfWeek.Monday };
+            yield return new object[] { "sr-Latn-RS", DayOfWeek.Monday };
+            yield return new object[] { "sv-AX", DayOfWeek.Monday };
+            yield return new object[] { "sv-SE", DayOfWeek.Monday };
+            yield return new object[] { "sw-CD", DayOfWeek.Monday };
+            yield return new object[] { "sw-KE", DayOfWeek.Sunday };
+            yield return new object[] { "sw-TZ", DayOfWeek.Monday };
+            yield return new object[] { "sw-UG", DayOfWeek.Monday };
+            yield return new object[] { "ta-IN", DayOfWeek.Sunday };
+            yield return new object[] { "ta-LK", DayOfWeek.Monday };
+            yield return new object[] { "ta-MY", DayOfWeek.Monday };
+            yield return new object[] { "ta-SG", DayOfWeek.Sunday };
+            yield return new object[] { "te-IN", DayOfWeek.Sunday };
+            yield return new object[] { "th-TH", DayOfWeek.Sunday };
+            yield return new object[] { "tr-CY", DayOfWeek.Monday };
+            yield return new object[] { "tr-TR", DayOfWeek.Monday };
+            yield return new object[] { "uk-UA", DayOfWeek.Monday };
+            yield return new object[] { "vi-VN", DayOfWeek.Monday };
+            yield return new object[] { "zh-CN", DayOfWeek.Monday  };
+            yield return new object[] { "zh-Hans-HK", DayOfWeek.Sunday };
+            yield return new object[] { "zh-SG", DayOfWeek.Sunday };
+            yield return new object[] { "zh-HK", DayOfWeek.Sunday };
+            yield return new object[] { "zh-TW", DayOfWeek.Sunday };
         }
 
         [Theory]
         [MemberData(nameof(FirstDayOfWeek_Get_TestData))]
-        public void FirstDayOfWeek(DateTimeFormatInfo format, DayOfWeek expected)
+        public void FirstDayOfWeek(DateTimeFormatInfo format, DayOfWeek expected, string cultureName)
+        {
+            Assert.True(expected == format.FirstDayOfWeek, $"Failed for culture: {cultureName}. Expected: {expected}, Actual: {format.FirstDayOfWeek}");
```

Comment:
nit: Can we call `FirstDayOfWeek` from `FirstDayOfWeekHybridGlobalization` (share the assert "logic" and message)?

Reply:
good idea